.github/workflows/ci.yml

```diff
@@ -236,96 +236,3 @@ jobs:
         
       - name: Stop sccache server
         run: sccache --stop-server || true
-  
```

Comment:
What is the motivation for removing code cov?

Reply:
Oops didn't mean to was just doing find and replace operations to update the version and must have messed up a regex.